mpf/core/utility_functions.py

```diff
@@ -102,8 +102,12 @@ def string_to_list(string: Union[str, List[str], None]) -> List[str]:
 
         """
         if isinstance(string, str):
-            # Convert commas to spaces, then split the string into a list
-            new_list = string.replace(',', ' ').split()
+            if "{" in string:
+                # Split the string on spaces/commas EXCEPT regions within braces
+                new_list = re.findall(r'([\w|-]+?\{.*?\}|[\w|-]+)', string)
+            else:
+                # Convert commas to spaces, then split the string into a list
+                new_list = string.replace(",", " ").split()
```

Comment:
I guess whitespace and/or comma is also not such a good idea, right? We should convert this into something more explicit (not necessarily now)

Reply:
It feels like an "it was good enough at the time" decisions, and generally speaking it's sufficient. I wouldn't want to change anything that would break existing configs, although accepting both commas and spaces is more complex than doing strict space separation.